lang/en.json

```diff
@@ -3,17 +3,17 @@
   "ACTOR.TypeMonster": "Monster",
   "ACTOR.TypeStronghold": "Stronghold",
 
-  "ITEM.TypeWeapon": "Weapon",
   "ITEM.TypeArmor": "Armor",
+  "ITEM.TypeBuilding": "Stronghold Building",
```

Comment:
I notice the translation strings now use a mix of `CamelCase` and `UPPER_CASE_WITH_UNDERSCORES`. We should be consistent.

This wasn't changed in this PR, though, so it's fine to leave it for later.

Reply:
I may be wrong here, but to my knowledge this is to do with Foundry calling the localization string as `"Type"`. Admittedly I have not tried `SCREAMING_SNAKE_CASE`. I copied the format from 5e.

Reply:
Aha, I'm sorry then! I didn't realize these strings were put together by Foundry. Then we're at least consistent with the strings we have control over, so that's all good. :+1: